### PR TITLE
Book Detail III - Structure

### PR DESCRIPTION
**Pull request by mcolombo87**
## Summary

A little bit of style.

## Screenshots

image

## Trello Card

https://trello.com/c/3F6xFajP/28-detalle-de-libro-iii-estructura.

## Blocked by 

#10

### CODE REVIEW

File: maquetado/scss/details.scss [hunk 1]
```diff
@@ -1,48 +1,60 @@
...
+    max-width: 309px;
     padding: 24px;
     width: 100%;
   }
+
+  .book {
+    flex-grow: 1;
+    max-height: 368px;
+    max-width: 588px;
+    padding: 24px 24px 0px 24px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: You can change to padding: 24px 24px 0

File: maquetado/scss/details.scss [hunk 2]
```diff
@@ -1,48 +1,60 @@
...
+
+    .book-details {
+      .book-item {
+        display: block;
+        font-size: $font-normal;
+        padding: 36px 0 0 0 ;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Change to padding-left: 36px

File: maquetado/scss/details.scss [hunk 3]
```diff
@@ -1,48 +1,60 @@
...
+      }
+      
+      .book-item-span {
+        color: $grey;
+        font-size: $font-normal;
+      }
+    }
+    
+    .book-title {
+      padding: 0px 8px 12px 0px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: When you set in 0 the px is not necessary

File: maquetado/scss/details.scss (hunk 2 above)
Comment (erikankodelia) [reply maquetado/scss/details.scss]: But if you want to make a distance between the children and the parent, so add padding to book-details.

File: maquetado/scss/details.scss (hunk 1 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: done!

File: maquetado/scss/details.scss (hunk 2 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: This is not the case, that padding is to separate each paragraph (A great advice anyway). However, change the padding to padding-top.

File: maquetado/scss/details.scss (hunk 3 above)
Comment (mcolombo87) [reply maquetado/scss/details.scss]: Done!

File: maquetado/scss/details.scss [hunk 4]
```diff
@@ -1,48 +1,60 @@
...
+    flex-grow: 1;
+    max-height: 368px;
+    max-width: 588px;
+    padding: 24px 24px 0;
+
+    .book-details {    
+      .book-item {
+        display: block;
+        font-size: $font-normal;
+        padding-top: 36px;
```
Comment (erikankodelia) [inline maquetado/scss/details.scss]: Change to margin-bottom: 36px;

Comment (mcolombo87) [reply maquetado/scss/details.scss]: done!

**Pull request merged**